presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -635,7 +635,7 @@ private static QueryError toQueryError(ExecutionFailureInfo executionFailureInfo
     private static <T> void writeJsonFile(Path outputPath, T object, JsonCodec<T> codec)
     {
         try {
-            Files.write(outputPath, codec.toJsonBytes(object));
+            Files.write(outputPath, (codec.toJson(object) + "\n").getBytes());
```

Comment:
Does it pretty-prints when you do `toJson` vs `toJsonBytes`?

Reply:
Generally we are trying to keep the output as compact as possible. 

You can pretty-print in in the console with the

> cat file.json | python3 -m json.tool | less

Reply:
`toJson` did pretty-prints, I verified by printing in unit test. I feel like for output file of this size, ease of readability is more important.